stage3/01-install-wifi-manager/01-run.sh

```diff
@@ -0,0 +1,17 @@
+
+cp -r "${BASE_DIR}/../EffectiveRange/poc/wifiManager" "${ROOTFS_DIR}/tmp/wifiManager"
+
+on_chroot << EOF
+
+cd /tmp/wifiManager
+
+# Package wifi-manager
+dpkg-buildpackage -us -uc -b
+
+# Install wifi-manager
+apt-get install /tmp/wifi-manager_0.0.1_all.deb
```

Comment:
Would it be possible to fire up a local apt repo and just list the package in the list of packages and the. At the end of the build just unconfigure the install apt repo?